client/state/plugins/installed/test/selectors.ts

```diff
@@ -1,5 +1,5 @@
 import deepFreeze from 'deep-freeze';
-import { pick } from 'lodash';
+import { omit, pick } from 'lodash';
```

Comment:
Is that true even for unit tests? I understand it for production code since it's one less library to go in the bundle, but for code like this it seems like lodash can be pretty helpful.

Reply:
I'd argue it's true for any code. We own all of our code, and a lodash usage that exists in tests can easily sneak into production code if someone is borrowing code from that test as an example. Ideally, we shouldn't be relying on Lodash, regardless of the context. We're aiming to fully get rid of Lodash at some point, see p4TIVU-9Bf-p2.